README.md

```diff
@@ -228,6 +229,19 @@ Set the size of the memory map, the default value is `1048576`(1MB).
 
 [Back to TOC](#table-of-contents)
 
+### lmdb_validation_tag
+
+**syntax:** *lmdb_validation_tag value;*
+
```

Comment:
What is the default? What happens if the directive is not set?

Reply:
The default value is null string, then this feature will be disabled, it will do nothing.
```
393     if (lcf->validation_tag.data == NULL) {
394         ngx_log_debug0(NGX_LOG_DEBUG_CORE, cycle->log, 0,
395                        "LMDB validation disabled");
396         return NGX_OK;
397     }
```

Reply:
Please document this in README.

Reply:
done

Reply:
done.